src/wallet/wallet.cpp

```diff
@@ -2909,7 +2909,7 @@ std::shared_ptr<CWallet> CWallet::Create(WalletContext& context, const std::stri
     if (args.IsArgSet("-discardfee")) {
         std::optional<CAmount> discard_fee = ParseMoney(args.GetArg("-discardfee", ""));
         if (!discard_fee) {
-            error = strprintf(_("Invalid amount for -discardfee=<amount>: '%s'"), args.GetArg("-discardfee", ""));
+            error = strprintf(_("Invalid amount for %s=<amount>: '%s'"), "-discardfee", args.GetArg("-discardfee", ""));
```

Comment:
This message should probably be deduplicated to one location.

Reply:
yeah, probably we should first move all the init args reads and validations to a separate file (`wallet_init.cpp` or something similar). Then cleanup all the string messages at once. So we don't continue mixing init stuff with wallet's internals.